packages/server/src/config/setup/SwaggerSetup.ts

```diff
@@ -17,14 +20,21 @@ type SwaggerConfig = Readonly<{
 }>;
 
 const defaultSwaggerDocumentConfig: SwaggerDocumentConfig = Object.freeze({
-  title: 'App example',
-  description: 'The app API description',
+  title: `${APP_NAME} API 🍲🍝🍜`,
+  description: `
+${APP_NAME} is an online food delivery system connecting restaurants with health-conscious users.
+The app automatically sums up all of the nutritional values helping you stick to your diet.
```

Comment:
To podliczanie wartości odżywczych może być tu w sumie trochę mylące, bo jeszcze tego nie mamy